core/src/OpenACC/Kokkos_OpenACC.cpp

```diff
@@ -23,7 +23,17 @@
 #include <impl/Kokkos_ExecSpaceManager.hpp>
 #include <impl/Kokkos_DeviceManagement.hpp>
 
+#if defined(KOKKOS_IMPL_ARCH_NVIDIA_GPU)
+#include <cuda_runtime.h>
+#elif defined(KOKKOS_ARCH_AMD_GPU)
+#define __HIP_PLATFORM_AMD__
```

Comment:
`hip_runtime_api.h` contains two implementations: one for AMD GPUs and the other for NVIDIA GPUs; the macro, `__HIP_PLATFORM_AMD__` is needed to choose AMD GPUs. (We put it here since we don't know what kind of side effects it will have if it becomes a global macro.)
Added a comment.